src/components/SelectComponent/SelectComponent.jsx

```diff
@@ -0,0 +1,19 @@
+import topicArrow from "../../assets/upside-down-arrow-grey.svg";
+import "./SelectComponent.scss";
+
+const SelectComponent = () => {
+
+  return (
+    <>
+    <label className="select__label" htmlFor="">Lesson Type</label>
+    <div className='select'>
+        <select className='select__dropdown' name="" id=""  img={topicArrow}>
+            <option className="select__dropdownoption" value="">Lesson</option>
+            <option className="select__dropdownoption" value="">Assessment Quiz</option>
```

Comment:
The name, id and values are all blank, do they need to have something inside or can they be removed?